apps/mobile/src/hooks/useDelegateCleanup.ts

```diff
@@ -0,0 +1,114 @@
+import { useCallback, useState, useMemo } from 'react'
```

Comment:
Should the `useDelegateCleanup` hook not go into the `useDelegateCleanup` folder inside hooks?